eng/common/pipelines/templates/steps/publish-blobs.yml

```
@@ -21,6 +23,8 @@ steps:
       -SASKey "${{ parameters.BlobSASKey }}"
       -Language "${{ parameters.TargetLanguage }}"
       -BlobName "${{ parameters.BlobName }}"
+      -PublicArtifactLocation  "${{ parameters.ArtifactLocation}}""
+      -RepoReplaceRegex "(https://github.com/${{ parameters.RepoId }}/(?:blob|tree)/)master(/.*)"
```

Based on my other comments I think we should update this regex to 
```suggestion
      -RepoReplaceRegex "(https://github.com/${{ parameters.RepoId }}/(?:blob|tree)/)master(/[^ ")]*)"
```

---

github link is not always ends up with quote. There are some other break symobols like `<>,` etc

---

Ok we should look at and see what is the best set of break characters based on your general usages and replacements in html as well as md files and add those to the char set.

---

Talked offline. Decided to have simple regex